src/api/flash/FocusCommands.js

```diff
@@ -30,6 +30,14 @@ export function FocusCommands(options) {
     let port = focus._port;
 
     if (devicePort && focusDeviceDescriptor) {
+      if (port?.isOpen) {
+        try {
+          await port.close();
+        } catch (_) {
+          /* ignore the error */
+        }
+      }
+      focus._port = undefined;
```

Comment:
I think this line just convinced me that we want an acccessor / mutator on focus for the port variable.